packages/rs-dpp/src/data_contract/document_type/class_methods/create_document_types_from_document_schemas/v0/mod.rs

```diff
@@ -14,9 +15,7 @@ impl DocumentTypeV0 {
         data_contract_id: Identifier,
         document_schemas: BTreeMap<DocumentName, Value>,
         schema_defs: Option<&BTreeMap<String, Value>>,
-        documents_keep_history_contract_default: bool,
-        documents_mutable_contract_default: bool,
-        documents_can_be_deleted_contract_default: bool,
```

Comment:
You can't remove this

Reply:
There are 4 fields of data contract config that are passing as independent params. I changed it to 1 param - config itself.